ENCODE_fastq_trimmer.py

```diff
@@ -79,7 +79,8 @@ def trim_file(url, connection, filename, size):
                 header = next(gzfile)
```

Comment:
There are import errors in this file, L95-96

Reply:
That's another issue, as it doesn't work on master either:
```
DN0a22f83f:pyencoded-tools keenan$ python ENCODE_fastq_trimmer.py 
Traceback (most recent call last):
  File "ENCODE_fastq_trimmer.py", line 167, in <module>
    main()
  File "ENCODE_fastq_trimmer.py", line 94, in main
    key = encodedcc.ENC_Key(args.keyfile, args.key)
NameError: name 'encodedcc' is not defined
```
The real question is who commented out encodedcc from this file in the first place and why?
<img width="162" alt="screen shot 2017-10-19 at 3 26 17 pm" src="https://user-images.githubusercontent.com/16053922/31797179-e17d2290-b4e1-11e7-8d8a-f6e4c7273291.png">